dists/scummvm.rc

```diff
@@ -29,7 +29,7 @@ shaders.dat            FILE    "gui/themes/shaders.dat"
 
 #include "dists/scummvm_rc_engine_data_core.rh"
 
-#ifdef USE_FLUIDSYNTH
+#if defined(USE_FLUIDSYNTH) || defined(USE_FLUIDLITE)
```

Comment:
And this one should also pertain to the Makefile.common commit.

Reply:
I split this into a separate commit